### DISCUSSION TITLE
HOW TO OBTAIN THEM?

### DISCUSSION THREAD

**Discussion by iamdark123** [mezotv/discord-badges#24] [General]
Yeah Well I wanted to know how can we get the Hypesquad events badge now?

**Comment by mezotv** [comment]
> Yeah Well I wanted to know how can we get the Hypesquad events badge now?

The hypesquad events program has been closed and stopped for a while, its no longer possible to get.

**Reply by iamdark123** [reply]
Ohh I thought it works only in custom locations. BTW thanks.

**Reply by mezotv** [reply]
Hey do you think you could you mark my reply as the answer to this so I can get this post closed?

**Reply by inkoscommissions** [reply]
can i obtain the badges troght this ?

**Reply by mezotv** [reply]
You can probably use the API to skill get them but no official way
